pegasus/sites.v3/code.org/public/playlab.haml

```diff
@@ -5,14 +5,14 @@ social:
   "twitter:card": photo
   "twitter:site": "@codeorg"
   "twitter:url": "http://code.org/playlab"
-  "twitter:image:src" : "https://code.org/images/playlab_original3.png"
+  "twitter:image:src" : "https://code.org/shared/images/courses/logo_tall_playlab.jpg"
```

Comment:
do we know whether /images and /shared have the same caching policy? maybe a question for @philbogle